packages/webpack-plugin/src/getUserTheme.ts

```diff
@@ -0,0 +1,40 @@
+import eval from "eval";
+import { buildSync } from "esbuild";
+import fs from "fs";
+import type { UserTheme } from "packages/sheet";
+
+let prevConfigText: string | undefined;
+let pervUserTheme: Partial<UserTheme> | undefined;
+
+export const getUserTheme = (configPath: string) => {
+  const configText = fs.readFileSync(configPath, "utf8");
+  const isChanged = configText !== prevConfigText;
+  prevConfigText = configText;
+
+  if (!isChanged) {
+    return pervUserTheme;
+  }
```

Comment:
If there are no changes between loaders, and between plugins and loaders, we are returning cached values. Please let me know if there  Please let me know if there is a more efficient way to cache.